cmd/plugin/builder/template/plugintemplates/github_workflow_release.yaml.tmpl

```diff
@@ -2,49 +2,42 @@ name: Release
 
 on:
   push:
-  tags:
-    - v*
-  branches: [ master ]
+    tags:
+      - v*
+    branches: [ main ]
 
 jobs:
 
   build:
-  name: Release
-  runs-on: ubuntu-latest
-  steps:
-
-  - name: Set up Go 1.x
-    uses: actions/setup-go@v2
-    with:
-      go-version: ^1.18
-      id: go
-
-  - name: Check out code into the Go module directory
-    uses: actions/checkout@v2
-
-  - name: Get dependencies
-    run: |
-    go get -v -t -d ./...
-    curl -o tanzu https://storage.googleapis.com/tanzu-cli/artifacts/core/latest/tanzu-core-linux_amd64 && \
-        mv tanzu /usr/local/bin/tanzu && \
-      chmod +x /usr/local/bin/tanzu
-    tanzu plugin repo add -b tanzu-cli-admin-plugins -n admin
-
-  - name: init
-    run: make init
-
-  - name: Build
-    run: make build
-
-  - name: Test
-    run: make test
-
-  - name: Test
-    run: make lint
-
-  - id: upload-cli-artifacts
-    uses: GoogleCloudPlatform/github-actions/upload-cloud-storage@master
-    with:
-      path: ./artifacts
-      destination: {{ .RepositoryName }}
-      credentials: {{"${{ secrets.GCP_BUCKET_SA }}"}}
+    name: Release
+    runs-on: ubuntu-latest
+    steps:
+
+    - name: Set up Go 1.x
+      uses: actions/setup-go@v2
+      with:
+        go-version: ^1.20
+        id: go
+
+    - name: Check out code into the Go module directory
+      uses: actions/checkout@v1
```

Comment:
changed to `actions/checkout@v1` instead of `actions/checkout@v2` because someone reported that the v2 version would not clone tags, but tags are needed by the build to set the version of the plugin